.github/workflows/publish.yml

```diff
@@ -58,19 +59,21 @@ jobs:
 
     - uses: actions/upload-artifact@v2
       with:
-        path: ${{ github.workspace }}/dist/*.tar.gz
+        name: dist
+        path: dist/*.tar.gz
 
   publish:
     needs: [build_source, build_binary_manylinux]
-    if: github.event.inputs.disable_publish == 0
+    if: github.event.inputs.disable_publish == 'false'
```

Comment:
It is indeed `'false`' and not `false`, I'm not sure why GitHub decide to do it this way.